backend/core/src/listings/entities/listing.entity.ts

```diff
@@ -566,6 +569,18 @@ class Listing extends BaseEntity {
   @ValidateNested({ groups: [ValidationsGroupsEnum.default], each: true })
   @Type(() => UnitsSummary)
   unitsSummary: UnitsSummary[]
+
+  @OneToOne(() => ListingFeatures, {
+    nullable: true,
+    eager: true,
+    cascade: true,
+  })
+  @JoinColumn()
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default], each: true })
+  @ValidateNested({ groups: [ValidationsGroupsEnum.default], each: true })
+  @Type(() => ListingFeatures)
+  features: ListingFeatures
```

Comment:
Should this be `features?`? I think then you could avoid adding `features: null` to all the seed scripts.

Reply:
Oh, that's way smarter, duh. Thanks!